src/ncp/ncp_base.cpp

```diff
@@ -1722,6 +1722,14 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CAPS>(void)
     SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_MCU_POWER_STATE));
 #endif
 
+#if OPENTHREAD_CONFIG_RADIO_2P4GHZ_OQPSK_SUPPORT
+    SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_802_15_4_2450MHZ_OQPSK));
```

Comment:
Remove the use of `SPINEL_CAP_802_15_4_2450MHZ_OQPSK` from L#1795 below?

Reply:
Thanks